Source/BugsnagConfiguration.m

```diff
@@ -185,10 +185,27 @@ - (void)setUserMetadataFromUser:(BugsnagUser *)user {
     [self.metadata addAttribute:BSGKeyEmail withValue:user.emailAddress toTabWithName:BSGKeyUser];
 }
 
```

Comment:
The `onSendBlocks` property is publicly accessible. This (and the other callback arrays) should be hidden as part of the internal implementation and accessed via a class extension or similar mechanism.

Reply:
Agreed.  Done for `onSend` and `onSession`.